src/main/resources/db/migration/V107__add_metadata_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE Metadata (
+    key VARCHAR NOT NULL UNIQUE,
```

Comment:
SonarCloud is complaining that VARCHAR does not have a size. It is a decent complaint honestly. I would put VARCHAR(255) on both fields.

Also I don't think there is a reason to not have a PRIMARY KEY on a table. I would set `key VARCHAR(255) PRIMARY KEY`

Reply:
As it was a not generated key value I thought on using `UNIQUE` instead of `PRIMARY KEY` but it makes more sense, I will change it